DirectProgramming/C++SYCL_FPGA/Tutorials/GettingStarted/experimental/fpga_template/sample.json

```diff
@@ -0,0 +1,94 @@
+{
+  "guid": "418AA4D8-0A61-4FDB-845A-CDA4DA0655E7",
+  "name": "FPGA Template",
+  "categories": ["Toolkit/oneAPI Direct Programming/DPC++ FPGA/Getting Started Tutorials"],
+  "description": "This project serves as a template for Intel® oneAPI FPGA designs. ",
+  "toolchain": ["dpcpp"],
+  "os": ["linux", "windows"],
+  "targetDevice": ["FPGA"],
+  "builder": ["cmake"],
+  "languages": [{"cpp":{}}],
+  "commonFolder": {
+    "base": "../../..",
+    "include": [
+      "README.md",
+      "Tutorials/GettingStarted/experimental/fpga_template",
```

Comment:
Does this sample still need to be experimental?

Reply:
the only reason to be 'experimental' was to hide IP Authoring. I'm still not very enamoured with the state of IP authoring (host pipes are still quite terrible). I'll discuss with duncan  what to do.

Reply:
moved out of experimental